raiden/network/pathfinding.py

```diff
@@ -298,7 +298,7 @@ def configure_pfs_or_exit(
         f"running version: {pathfinding_service_info.version}, "
         f"chain_id: {pathfinding_service_info.chain_id}.\n"
         f"Fees will be paid to {to_checksum_address(pathfinding_service_info.payment_address)}. "
-        f"For each request costs {pathfinding_service_info.price}.\n"
+        f"Each request costs {pathfinding_service_info.price} RDN.\n"
```

Comment:
Is the RDN or RAI?

Reply:
Good question. You are the best person to answer this since this is taken directly from the PFS. But loking at the code I would expected RAI. Right? So I will have to make an adjustment here.

Reply:
Yes, it's RAI. Thanks for the fix!